content/en/apps/tutorials/localizing-cht.md

```diff
@@ -0,0 +1,232 @@
+To localize instance text to Swahili, change the default system language to `Swahili`.
+
+Go to App Management > Display > Languages > Default Language(Change to Swahili)
+
+![configuration](change-system-language.png)
+
+Create a `message-sw.properties` file and use the instructions outlined [here]({{< ref "../reference/translations/#translations" >}}) to learn the structure of a message-{language-code}.properties file.
+
+Populate the `messages-sw.properties` file with the appropriate translation strigs and upload it using the below command:
+
+```
+cht --local upload-custom-translations
+```
+The default Swahili translations that come pre-added to CHT can be found [here](https://github.com/medic/cht-core/blob/master/ddocs/medic/_attachments/translations/messages-sw.properties).
+
+After changing the instance language to Swahili, the various elements will behave like this:
+
+_**Messages**_
+
+In Messages, the time counter text and navigation text changes.
+
+See below example for Swahili localization.
+
+*English text*
+
+![configuration](messages-tab-language-en.png)
+
+*Swahili Translation*
+
+![configuration](messages-tab-language-sw.png)
+
+_**Tasks**_
+
+You can localize the task header by adding the appropriate translation in the `messages-sw.properties` file.
+
+e.g to translate the below delivery task title to Swahili
+```
+{
+    name: 'anc-home-visit-delivery',
+    icon: 'icon-pregnancy',
+    title: 'task.anc.delivery.title',
+``` 
+
+Add the below code to the `messages-sw.properties` file
+```
+task.anc.delivery.title = Kazi ya Kujifungua
+```
+
+*Default English  text*
+
+![configuration](localize-tasks-en.png)
+
+*Swahili translation*
+![configuration](localize-tasks-sw.png)
+
+_**Reports**_
+
+You can localize the report field names by adding the appropriate translation in the `messages-{language-code}.properties` file.
+
+e.g to change the date of birth field to Swahili, in `messages-sw.properties` file, add this:
+
+```
+contact.type.date_of_birth = Siku ya Kuzaliwa
+```
+*Before*
+
+![configuration](localize-reports-en.png)
+
+*Swahili Translation*
+
+![configuration](localize-reports-sw.png)
+
+
+e.g 
+
+_**People**_
+
+To localize the contact labels, add the appropriate translation in `messages-{language-code}.properties` file.
+
+e.g to change the people name label translation from the default Swahili translation of `Watu` to `Watu wa hili hapa eneo`, in `messages-sw.properties` file, add this:
+
+```
+contact.type.person = Mtu wa hili hapa eneo
+contact.type.person.plural = Watu wa hili hapa eneo
+```
+
+*Before*
+
+![configuration](people-translation-en.png)
+
+*After
+
+![configuration](people-translation-sw.png)
+
+_**Targets**_
+
+You can localize the names of the targets by adding the appropriate translation in the `messages-{language-code}.properties file.
+
+e.g to add the `Growth Monitoring` target title in Swahili on the instance, add the appropriate translation in the `messages-sw.properties` file. i.e
```

Comment:
```suggestion
For example, to add the `Growth Monitoring` target title in Swahili on the instance, add the appropriate translation in the `messages-sw.properties` file. For instance:
```
Avoid abbreviations whenever you can. Use for example instead of e.g., and for instance instead of i.e.